src/plots/plots.js

```diff
@@ -257,6 +257,8 @@ plots.previousPromises = function(gd) {
  * Add source links to your graph inside the 'showSources' config argument.
  */
 plots.addLinks = function(gd) {
+    if(gd._context.showLink === false) return;
```

Comment:
This could break graphs that have the [`showSources`](https://github.com/plotly/plotly.js/blob/ce06ea9cea377c5ed87ed7baa088a14a1997dbb7/src/plot_api/plot_config.js#L67) config option set as it is called [below](https://github.com/plotly/plotly.js/blob/ce06ea9cea377c5ed87ed7baa088a14a1997dbb7/src/plots/plots.js#L308). 

So, replacing this line with something like:

```js
if(!gd._context.showLink || !gd._context.showSources) return;
```

would be safer.